test/framework/functional/vector/deploy.go

```diff
@@ -13,7 +15,7 @@ import (
 )
 
 const entrypointScript = `#!/bin/bash
```

Comment:
Doesn't this get replaced by the content of run_script.go

Reply:
no, in working environment launch script mount to the` /usr/bin/run-vector.sh`, in test environment` /opt/app-root/src/run.sh `